Adyen/client.py

```diff
@@ -485,107 +483,82 @@ def _handle_http_error(self, url, response_obj, status_code, psp_ref,
-            else:
-                erstr = "Unexpected error while communicating with Adyen." \
-                        " Please reach out to support@adyen.com" \
-                        " if the problem persists"
-                raise AdyenAPICommunicationError(erstr,
-                                                 raw_request=raw_request,
-                                                 raw_response=raw_response,
-                                                 url=url,
-                                                 psp=psp_ref,
-                                                 headers=headers,
-                                                 error_code=response_obj.get(
-                                                     "errorCode"))
+            raise AdyenAPICommunicationError(message,
+                                             raw_request=raw_request,
+                                             raw_response=raw_response,
+                                             url=url,
+                                             psp=psp_ref,
+                                             headers=headers,
+                                             status_code=status_code,
+                                             error_code=response_obj.get(
+                                                 "errorCode"))
         elif status_code == 400:
-            erstr = "Received validation error with errorCode: %s," \
-                    " message: %s, HTTP Code: %s. Please verify" \
-                    " the values provided. Please reach out" \
-                    " to support@adyen.com if the problem persists," \
-                    " providing the PSP reference: %s" % (
-                        response_obj["errorCode"], response_obj["message"],
-                        status_code, psp_ref)
-
-            raise AdyenAPIValidationError(erstr, error_code=response_obj.get(
-                "errorCode"))
+            raise AdyenAPIValidationError(message,
+                                          raw_request=raw_request,
+                                          raw_response=raw_response,
+                                          url=url,
+                                          psp=psp_ref,
+                                          headers=headers,
+                                          status_code=status_code,
+                                          error_code=response_obj.get(
+                                              "errorCode"))
         elif status_code == 401:
-            erstr = "Unable to authenticate with Adyen's Servers." \
-                    " Please verify the credentials set with the Adyen base" \
-                    " class. Please reach out to your Adyen Admin" \
-                    " if the problem persists"
-            raise AdyenAPIAuthenticationError(erstr,
+            raise AdyenAPIAuthenticationError(message,
+                                              raw_request=raw_request,
```

Comment:
can we do `raw_response,` instead of `raw_response=raw_response,`? we do that for message as well right? Might be easier to read and more consistent

Reply:
Sure! Changed all of them, also moved the retrieving of the error code (if exists) to the beginning as it is reused in multiple lines